src/surfaces/actionBar/actionBarActionsButtonDrawerSubOption.jsx

```diff
@@ -99,6 +99,7 @@ class ActionBarActionsButtonDrawerSubOption extends React.PureComponent {
         return (
             <div
                 className={subOptionClasses}
+                data-testid="actions_button_drawer--sub_option"
```

Comment:
See my note above about `data=testid` on the corresponding option component

Reply:
Fixed